README.md

```diff
@@ -79,6 +79,7 @@ The only required property is `value`.
 Option | Type | Description | Default
 -------|------|-------------|--------
 value | Number | Use the `v-model` attribute to set the value of the control | none
+animation | Object | Animation config | {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>}
```

Comment:
Move this table entry back to the bottom, also change what's in the default column.

Reply:
Ok i’ll Move it back to the bottom. 

How do you want to write this section ?
It is an object with default, so entire object should be in the cell.

Reply:
I would move the example object into the description column, like this:

animation | Object | Optional animation config object: {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>} | null